### ISSUE TITLE
Can't Install  DownZemAll on Manjaro 21

### ISSUE THREAD

**Issue by justaordinaryman** [setvisible/DownZemAll#74]
I Can't install on Manjaro 21 

i install DownZemAll from aur package but always get an error when installing  [AUR](https://aur.archlinux.org/packages/downzemall)
when i tried .appimage version, i got same error as #60 

Error Log :
`Test project /home/tiko/.cache/yay/downzemall/src/build
      Start  1: tst_attribute
 1/23 Test  #1: tst_attribute ....................   Passed    0.07 sec
      Start  2: tst_charref
 2/23 Test  #2: tst_charref ......................   Passed    0.02 sec
      Start  3: tst_parser
 3/23 Test  #3: tst_parser .......................   Passed    0.03 sec
      Start  4: tst_stringbuffer
 4/23 Test  #4: tst_stringbuffer .................   Passed    0.03 sec
      Start  5: tst_stringpiece
 5/23 Test  #5: tst_stringpiece ..................   Passed    0.02 sec
      Start  6: tst_tokenizer
 6/23 Test  #6: tst_tokenizer ....................Subprocess aborted***Exception:   0.63 sec
********* Start testing of tst_Tokenizer *********
Config: Using QtTest library 5.15.4, Qt 5.15.4 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 12.1.0), manjaro unknown
PASS   : tst_Tokenizer::initTestCase()
munmap_chunk(): invalid pointer

      Start  7: tst_utf8
 7/23 Test  #7: tst_utf8 .........................   Passed    0.14 sec
      Start  8: tst_vector
 8/23 Test  #8: tst_vector .......................   Passed    0.16 sec
      Start  9: tst_abstractsettings
 9/23 Test  #9: tst_abstractsettings .............   Passed    0.15 sec
      Start 10: tst_downloadengine
10/23 Test #10: tst_downloadengine ...............   Passed    2.60 sec
      Start 11: tst_downloadmanager
11/23 Test #11: tst_downloadmanager ..............***Failed    5.02 sec
********* Start testing of tst_DownloadManager *********
Config: Using QtTest library 5.15.4, Qt 5.15.4 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 12.1.0), manjaro unknown
PASS   : tst_DownloadManager::initTestCase()
FAIL!  : tst_DownloadManager::appendJobPaused() Compared values are not the same
   Loc: [/home/tiko/.cache/yay/downzemall/src/DownZemAll-2.5.7/test/core/downloadmanager/tst_downloadmanager.cpp(109)]
PASS   : tst_DownloadManager::cleanupTestCase()
Totals: 2 passed, 1 failed, 0 skipped, 0 blacklisted, 4969ms
********* Finished testing of tst_DownloadManager *********

      Start 12: tst_fileutils
12/23 Test #12: tst_fileutils ....................   Passed    0.06 sec
      Start 13: tst_format
13/23 Test #13: tst_format .......................   Passed    0.02 sec
      Start 14: tst_mask
14/23 Test #14: tst_mask .........................   Passed    0.03 sec
      Start 15: tst_regex
15/23 Test #15: tst_regex ........................   Passed    0.01 sec
      Start 16: tst_resourceitem
16/23 Test #16: tst_resourceitem .................   Passed    0.03 sec
      Start 17: tst_stream
17/23 Test #17: tst_stream .......................   Passed    0.04 sec
      Start 18: tst_torrentbasecontext
18/23 Test #18: tst_torrentbasecontext ...........   Passed    0.03 sec
      Start 19: tst_torrentcontext
19/23 Test #19: tst_torrentcontext ...............   Passed    0.09 sec
      Start 20: tst_updatechecker
20/23 Test #20: tst_updatechecker ................   Passed    0.11 sec
      Start 21: tst_jsonhandler
21/23 Test #21: tst_jsonhandler ..................   Passed    0.02 sec
      Start 22: tst_texthandler
22/23 Test #22: tst_texthandler ..................   Passed    0.02 sec
      Start 23: tst_interprocesscommunication
23/23 Test #23: tst_interprocesscommunication ....   Passed    0.01 sec

91% tests passed, 2 tests failed out of 23

Total Test time (real) =   9.40 sec

The following tests FAILED:
      6 - tst_tokenizer (Subprocess aborted)
     11 - tst_downloadmanager (Failed)
Errors while running CTest`

**Comment by setvisible** [comment]
Bug successfully reproduced in appveyor and travis !

Lol it's due to a ridiculous test:

```
11: ********* Start testing of tst_DownloadManager *********
11: Config: Using QtTest library 5.6.2, Qt 5.6.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 5.4.0 20160609)
11: PASS   : tst_DownloadManager::initTestCase()
11: FAIL!  : tst_DownloadManager::appendJobPaused() 'spyJobFinished.wait(5000)' returned FALSE. (
11: 
11: Connection Timeout
11: Can't reach:
11: http://preparer-assr.education-securite-routiere.fr/images/logo-securite-routiere.png
11: 
11: )
11:    Loc: [/home/travis/build/setvisible/DownZemAll/test/core/downloadmanager/tst_downloadmanager.cpp(105)]
11: PASS   : tst_DownloadManager::cleanupTestCase()
11: Totals: 2 passed, 1 failed, 0 skipped, 0 blacklisted
11: ********* Finished testing of tst_DownloadManager *********
11/23 Test #11: tst_downloadmanager ..............***Failed    5.28 sec
```

:o)

**Comment by setvisible** [comment]
Fixed in 2.5.8

**Comment by candrapersada** [comment]
how to fix ` "This application failed to start because it could not find or load the Qt platform plugin "xcb""` for appimage?
